style.css

```
@@ -6,13 +6,50 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
+    transition: color-scheme 0.7s ease,background-color 0.7s ease-in-out;
+
+}
+:root{
+    --primary-color: #edf2fc;
+    --secondary-color: #212121;
+    --tery-color: rgb(202, 196, 223);
+    --arrr: drop-shadow(5px 2px 7px #5c5c61);
+    /* --primary-color: #4f4c4ce9;
+    --secondary-color: #fff; 
+    --tery-color: rgb(139, 127, 185);
+    color-scheme: dark;  */
```

```suggestion
    /* --primary-color: #4f4c4ce9;
    --secondary-color: #fff; 
    --tery-color: rgb(139, 127, 185);
    color-scheme: dark;  */
```
Remove the unwanted content.